ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```
@@ -160,7 +160,19 @@ images:
 - context_dir: jupyter/minimal/ubi9-python-3.9
   dockerfile_path: Dockerfile
   from: amd-c9s-python-3.9
-  to: amd-jupyter-minimal-c9s-python-3.9
+  to: amd-jupyter-minimal-ubi9-python-3.9
```

```suggestion
  to: amd-jupyter-minimal-c9s-python-3.9
```


Even though we use the directory, ubi9-python-3.9
we are still basing these on centos , so we should use the centos in naming them
can you do this for the below ones as well.